images/jdk/image.yaml

```diff
@@ -10,4 +10,16 @@ versions:
       config: configs/openjdk-11.apko.yaml
       extractTagsFrom:
         package: openjdk-11
+        prefix: openjdk-
+  - apko:
+      config: configs/latest-dev.apko.yaml
+      extractTagsFrom:
+        package: openjdk-17-dev
```

Comment:
There doesn't appear to be a Wolfi package named `openjdk-17-dev`, I think we want this to be `package: openjdk-17`, and `prefix: dev-`?

That will give us `:dev-openjdk-17-11.0.18-r1`, is that right?

Reply:
fixed

Reply:
I think @imjasonh comment is still valid, there isn't a package called `openjdk-17-dev` should this be `openjdk-17`?